app/scripts/components/app/add-raw.js

```diff
@@ -48,13 +49,13 @@ const AddRaw = React.createClass({
     }
   },
 
-  componentWillMount: function () {
+  UNSAFE_componentWillMount: function () {
     if (this.props.defaultValue) {
       this.setState({ data: JSON.stringify(this.props.defaultValue, null, 2) });
     }
   },
 
-  componentWillReceiveProps: function ({ state }) {
+  UNSAFE_componentWillReceiveProps: function ({ state }) {
```

Comment:
It seems like the recommendation here is to upgrade to `componentDidUpdate`: https://reactjs.org/docs/react-component.html#unsafe_componentwillmount. Though the parameters of the function are different and some of the code running inside in these methods would need to be re-factored